src/app/editor/components/node-editor/node-editor.scss

```diff
@@ -19,7 +19,7 @@
 
 .title-row {
     display: flex;
-    padding: 5px;
+    padding: $gtx-block-padding;
```

Comment:
use `gtx-` prefix only for UI core variables. Use `meshui` for mesh ui variables.